pkg/gce/gce.go

```diff
@@ -103,7 +103,7 @@ func NewContext(customZoneID string) (*Context, error) {
 }
 
 func (ctx *Context) CreateInstance(name, machineType, image, sshkey string,
-	preemptible, displayDevice bool) (string, error) {
+	preemptible, displayDevice, nestedVirt bool) (string, error) {
```

Comment:
Looks like it's already the time to create a structure with arguments and pass that structure to the function instead of adding new ones.